script/passenger-recycler.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+#
+# Trivial Passenger memory monitor installed via /etc/cron.hourly/. To modify
+# configuration create /etc/foreman/passenger-recycler.rb.conf file.
+#
+
+CONFIG = '/etc/foreman/passenger-recycler.rb.conf'
```

Comment:
I know that we are using ENV in [foreman-tasks](https://github.com/theforeman/foreman-tasks/blob/master/deploy/foreman-tasks.sysconfig). Plus we can set ENV inline for use cases like cron:

```
GRACEFUL_SHUTDOWN_SLEEP = 1 passenger-recycler.rb
```
I don't have strong opinion here though, it's a question for people with actual administration experience.